test/fixtures/single-account-ecs/main.tf

```diff
@@ -4,6 +4,10 @@ terraform {
       source  = "sysdiglabs/sysdig"
       version = ">=0.5.33"
     }
+    aws = {
+      source  = "hashicorp/aws"
+      version = "<4.51.0"
+    }
```

Comment:
let's clarify that this version has been pinned due to an error with latest aws version, reported by @tembleking 

https://github.com/hashicorp/terraform-provider-aws/pull/29042